sta_exercise_final.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    for x in d['FFspksR'][0]:
</pre><p>Absolutely not necessary to get rid of the '-1's. You can histogram to produce the PSTH and they are out of range so go away on their own.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NeuroTheoryUMD/mattjac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>After discussing, this is not necessary for this analysis, b/c be are binning all the trials together, rather than looking at individual trials.</p>